packages/normalizr/src/schemas/Array.ts

```diff
@@ -43,20 +43,15 @@ export const denormalize = (
   unvisit: any,
 ): [denormalized: any, found: boolean, suspend: boolean] => {
   schema = validateSchema(schema);
-  let deleted = false;
-  let found = true;
-  if (input === undefined && schema) {
-    [, found, deleted] = unvisit(undefined, schema);
-  }
   return [
-    input && input.map
+    input.map
```

Comment:
are you expecting a non-array here with `map` present? or could this use `Array.isArray`